kairon/api/app/routers/bot/bot.py

```
@@ -1670,3 +1671,16 @@
     """
     data = MailProcessor.get_log(current_user.get_bot(), start_idx, page_size)
     return Response(data=data)
+
+
+@router.get("/mail_channel/read_mailbox", response_model=Response)
+async def trigger_mail_channel_read(
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS),
```

Ruff (0.8.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)